pandas/tests/generic/test_frame.py

```diff
@@ -280,3 +281,30 @@ def test_deepcopy_empty(self):
         empty_frame_copy = deepcopy(empty_frame)
 
         self._compare(empty_frame_copy, empty_frame)
+
+    def test_datetime_after_setitem_with_at(self):
```

Comment:
this should go in tests.indexing.test_scalar

is "at" the function being tested?  if so, the name should be something like "test_at_with..."